src/utils/oneDimensionalNoise/index.js

```diff
@@ -3,31 +3,40 @@
 // Must be power of 2 to work with bitwise modulo
 const MAX_VERTICES = 512
 const MAX_VERTICES_MASK = MAX_VERTICES - 1
+const BUFFERS_COUNT = 'abcdefghijklmnopqrstuvwyxz'.length
 
 function lerp (a, b, t) {
   return a * (1 - t) + b * t
 }
 
 class OneDimensionalNoise {
-  constructor (scale) {
-    this.buffer = []
-    this.call = 0
-
-    for (let i = 0; i < MAX_VERTICES; ++i) {
-      this.buffer.push(Math.random())
+  constructor () {
+    this.buffers = []
+
+    for (let i = 0; i < BUFFERS_COUNT; ++i) {
+      this.buffers[i] = []
+      for (let j = 0; j < MAX_VERTICES; j++) {
+        this.buffers[i].push(Math.random())
+      }
     }
   }
 
-  getValue (x) {
+  getValue (x, id) {
     const xFloor = Math.floor(x)
     const t = x - xFloor
     const tRemapSmoothstep = t * t * (3 - 2 * t)
+    const bufferId = id.charCodeAt(0) & BUFFERS_COUNT
+
+    let offset = 0;
+    for (let i = 1; i < id.length; i++) {
+      offset += id.charCodeAt(i)
+    }
```

Comment:
There's a 1/25th chance that noises share the same buffer but the offset will differentiate them. :)